coalib/bearlib/languages/definitions/Python.py

```diff
@@ -13,3 +13,4 @@ class Python:
     multiline_string_delimiters = {'"""': '"""', "'''": "'''"}
     indent_types = ':',
     encapsulators = {'(': ')', '[': ']', '{': '}'}
+    indent_keywords = 'if', 'while', 'for'
```

Comment:
Does `class` and `def` should be included here too?

Reply:
What about `else` and `with`?